vita3k/host/include/host/functions.h

```diff
@@ -22,7 +22,7 @@
 
 struct SfoFile;
 
-void install_pup(const std::string &pref_path, const std::string &pup_path);
+void install_pup(const std::string &pref_path, const std::string &pup_path, uint32_t *progress = 0);
```

Comment:
I think it's more clean to have a callback function that reports the progress so that the caller can manage the race condition things, instead of requiring pointer.